integration-tests/tests/ubuntuFan_test.go

```diff
@@ -1,5 +1,5 @@
 // -*- Mode: Go; indent-tabs-mode: t -*-
-// +build !excludeintegration
+// +build !excludeintegration,!excludeflaky
```

Comment:
Understood, but it's still not clear to me _why_ is it flaky in a way that makes us comfortable with that. Can't we fix the test, instead of avoiding it?  Unreliable tests in low performance generally means timing issues, which generally means not waiting long enough for events that we should be waiting for.

As an unrelated aside, such a tag might be simply "lowperformance", with the statement being "this is a low performance system". It'd be up to the expression to tell what to do with that. But I'd still prefer to avoid that altogether.